package.json

```diff
@@ -130,5 +129,8 @@
   "workspaces": [
     "modules/**"
   ],
-  "browserslist": "> 5%, last 2 firefox versions, last 2 chrome versions, last 2 safari versions, last 2 edge versions, ie 11"
+  "browserslist": "> 5%, last 2 firefox versions, last 2 chrome versions, last 2 safari versions, last 2 edge versions, ie 11",
+  "dependencies": {
+    "commitlint": "^8.1.0"
```

Comment:
Can you change this to a dev dependency, please? All dependencies for the top level monorepo package.json should be development ones.